driver/controller.go

```diff
@@ -23,7 +23,7 @@ import (
 	"strings"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
-	"github.com/gig-tech/ovc-sdk-go/ovc"
+	"github.com/gig-tech/ovc-sdk-go/v2/ovc"
```

Comment:
Shouldn't this be changed only once in `go.mod`, leaving the actual imports unmodified?

Reply:
Not for mayor versions

Reply:
https://medium.com/@elliotchance/major-versions-in-go-modules-explained-ec7c1df3888b